test/range.js

```diff
@@ -25,4 +26,15 @@ describe('range', function() {
     result.push(5);
     assert.deepEqual(R.range(10, 5), []);
   });
+
+  it('terminates given bad input', function() {
+    assert.throws(
+      function() { R.range('a', 'z'); },
+      function(err) {
+        return err.constructor === TypeError &&
+               err.message === 'Both arguments to range must be numbers';
```

Comment:
I really believe this style of testing the specific message in a unit-test is over-specification.

Checking that we throw a TypeError should be plenty.